reto.exs

```diff
@@ -0,0 +1,23 @@
+defmodule Recursion do
+  def print_multiple_times(msg, cantidad, cont, limit_inf, limit_sup) when cont <= limit_sup do
```

Comment:
Same with the method names, it could be something like count_divisible_numbers for instance 😄